src/duplicacy_b2client.go

```diff
@@ -81,7 +81,7 @@ func (client *B2Client) retry(backoff int, response *http.Response) int {
 	return backoff
 }
 
-func (client *B2Client) call(url string, input interface{}) (io.ReadCloser, int64, error) {
+func (client *B2Client) call(url string, requestHeaders map[string]string, input interface{}) (io.ReadCloser, http.Header, int64, error) {
```

Comment:
It may be better to add the method as an argument and not pass requestHeaders.

Reply:
Is there any desire to keep requestHeaders as an argument in the event that other calls need to pass anything in the future? Looking at the B2 API it looks like only b2_upload_file and b2_download_file_by_name take a request header other than authorization.

I'll make the method an enum and construct the header in that switch.